routes/index.js

```diff
@@ -2,8 +2,57 @@ var express = require('express');
 var passport = require('passport');
 var httpProxy = require('http-proxy');
 var ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn()
+var crypto = require('crypto');
 var router = express.Router();
 
+
+function formatDate(d) {
```

Comment:
I wonder if we should use something like [moment.js](http://momentjs.com/docs/#/displaying/), but feel free to leave this as it is.

(I played with it a bit in [jsfiddle here](https://jsfiddle.net/995tbe42/1/) if interested, but that's NOT TESTED, so not sure if it would still do what you're doing here)